### ISSUE TITLE
You must call one of in() or append() methods before iterating over a Finder

### ISSUE THREAD

**Issue by best-username** [OskarStark/php-cs-fixer-ga#80]
![image](https://github.com/OskarStark/php-cs-fixer-ga/assets/35339682/c8ba3ce5-83c7-424a-92ba-4a78b7ce564d)

Each time when do pull request or push I get this error. How to fix?

**Comment by OskarStark** [comment]
You should check your php-cs-fixer config

**Comment by criskell** [comment]
> You should check your php-cs-fixer config

Hello, I'm getting the same error, I'm not using a configuration file, I'm using the PHP-Cs-Fixer default settings
In the OP's screenshot, it looks like he isn't using it either, according to the message "Loaded config default".

**Comment by best-username** [comment]
@criskell  you are right. it is action for github, I have not change any conf files, so @OskarStark help please

**Comment by OskarStark** [comment]
Can you please try to add a config file please?

**Comment by browniebroke** [comment]
Had the same issue, fixed it by creating a config, as suggested. In practical terms, I created the following file:

```php
// .php-cs-fixer.dist.php
<?php

$finder = PhpCsFixer\Finder::create()->in('.');
$config = new PhpCsFixer\Config();
return $config->setFinder($finder);
```
